### PR TITLE
Created Power Daredevil

### PR DESCRIPTION
**Pull request by ScoobySharky**
Created Power Daredevil (still need to add the build but i have no access to GW2 right now)

### CODE REVIEW

File: content/builds/power-daredevil.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+tags = ["power","thief","daredevil","hot","lowcog","lowrep","lowphys","raid","strike"]
+authors = ["scooby"]
+
+[extra]
+series = "daredevil"
+tagline = "Let me smash your head in with this sparkly magic staff"
+keywords = "Guild Wars 2, GW2, LI, Daredevil"
+toc = true
+spec = "daredevil"
+balance = "March 2024"
```
Comment (berdandy) [inline content/builds/power-daredevil.md:19]: Now April 2024

File: content/builds/power-daredevil.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- Full Berserker's, with Runes of the Scholar or Dragonhunter
+- Staff with Force/Impact Sigils
+- Relic of the Thief
+- Consumables:
+  - Cilantro Lime Sous-Vide Steak (Power/Ferocity)
+  - Superior Sharpening Stone (Power)
+
+## Build
+
+TBC (I have no access to GW2 rn)
```
Comment (berdandy) [inline content/builds/power-daredevil.md:35]: Obviously this must be present.

File: content/builds/power-daredevil.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
++++
+title = "Power Daredevil"
+description = "Very high damage for very little APM, but requires great positioning."
+date = 2024-04-15
```
Comment (berdandy) [inline content/builds/power-daredevil.md:4]: Update to current date when build code becomes available

**Review by berdandy — CHANGES_REQUESTED** [review]
Looks ok. Needs the chat code.

**Comment by ScoobySharky** [comment]
Have fixed all the issues, and included the chat code!

**Review by berdandy — APPROVED** [review]

**Pull request merged**